ckanext/canada/templates/recombinant/create_pd_record.html

```diff
@@ -66,7 +66,7 @@
                 'required': required,
                 'help_text': description,
                 'choices': choice_fields[f.datastore_id],
-                'sorted_choices': true,
+                'sorted_choices': false if f.choice_order_expression else true,
```

Comment:
@RabiaSajjad you will need to do this in the `update_pd_record.html` template as well

Reply:
@JVickery-TBS committed this change to PR